frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeSidebar.unit.spec.tsx

```diff
@@ -225,26 +215,88 @@ describe("SummarizeSidebar", () => {
   });
 
   it("should list breakouts added before opening the sidebar in a separate section", async () => {
-    await setup({ card: createSummarizedCard() });
-
-    const pinnedColumnList = screen.getByTestId("pinned-dimensions");
-    const unpinnedColumnList = screen.getByTestId("unpinned-dimensions");
+    await setup({ query: createSummarizedQuery() });
 
     expect(
-      within(pinnedColumnList).getByText("Product → Category"),
+      within(getPinnedColumnList()).getByText("Product → Category"),
     ).toBeInTheDocument();
     expect(
-      within(pinnedColumnList).getByText("Created At"),
+      within(getPinnedColumnList()).getByText("Created At"),
     ).toBeInTheDocument();
 
     expect(
-      within(unpinnedColumnList).queryByText("Category"),
+      within(getUnpinnedColumnList()).queryByText("Category"),
     ).not.toBeInTheDocument();
 
     // "Product → Created At" and "User → Created At" should still be there
-    expect(within(unpinnedColumnList).getAllByText("Created At")).toHaveLength(
-      2,
+    expect(
+      within(getUnpinnedColumnList()).getAllByText("Created At"),
+    ).toHaveLength(2);
+  });
+
+  it("should show mulitple breakouts of the same column", async () => {
+    await setup({
+      query: createQueryWithBreakoutsForSameColumn(),
+    });
+    expect(
+      within(getPinnedColumnList()).getAllByText("Created At"),
+    ).toHaveLength(2);
+  });
+
+  it("should allow to modify temporal units for breakouts of the same column", async () => {
+    const { getNextBreakouts } = await setup({
+      query: createQueryWithBreakoutsForSameColumn(),
+    });
+
+    await userEvent.click(await screen.findByText("by year"));
+    await userEvent.click(await screen.findByText("Quarter"));
+    expect(getNextBreakouts()).toMatchObject([
+      { displayName: "Created At: Quarter" },
+      { displayName: "Created At: Month of year" },
+    ]);
+
+    await userEvent.click(await screen.findByText("by month of year"));
+    await userEvent.click(await screen.findByText("Quarter of year"));
+    expect(getNextBreakouts()).toMatchObject([
+      { displayName: "Created At: Quarter" },
+      { displayName: "Created At: Quarter of year" },
+    ]);
+  });
+
+  it("should allow to remove breakouts of the same column", async () => {
```

Comment:
What about adding multiple breakouts on the same column in the Summarize sidebar ([Slack](https://metaboat.slack.com/archives/C0645JP1W81/p1723105848301859?thread_ts=1723071609.486599&cid=C0645JP1W81))?

Reply:
We do not allow that by design